recipe/meta.yaml

```diff
@@ -21,13 +25,19 @@ package:
 source:
   - url: https://github.com/NVIDIA/nvmath-python/archive/refs/tags/v{{ ".".join(version.split(".")[:3]) }}.tar.gz
     sha256: 155b60e6dbacfea617aa9a2e92d262062916ccc2e3b51af199d4c1e2d9ef6555
+    patches:
+      - 000-curand-path.patch
 
 build:
   number: {{ number }}
-  skip: true  # [py < 310]
-  skip: true  # [osx or ppc64le]
+  skip: true  # [py < 310 or osx or ppc64le]
 
 requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cuda') }}  # [cuda_compiler_version != "None"]
+    - {{ compiler('cxx') }}
+    - {{ stdlib('c') }}
```

Comment:
Q: I assume this is to fix some quirky conda-build behaviors? (I dunno what, but it's why we had to add `host: - python` below...)

Reply:
Actually, it's conda-smithy that doesn't do a good job of detecting what build variants you need unless you put certain packages in the top-level requirements. I should probably add a note about this.

Reply:
Added a note about this.